Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,22 @@
 import React from 'react';
 import './Header.css';
+
+function HeaderTitle() {
+    return (
+        <div className="head">
+            <div className="head-title">
+                <h2>Lambda School</h2>
+                <a>@LambdaSchool</a>
+                <p>26 Jan</p>
```

Comment:
These should't be hard coded. These should be coming down in your `props` object and passed down from the parents.